.github/workflows/convention-develocity-gradle-plugin-verification.yml

```diff
@@ -3,10 +3,10 @@ name: Verify Convention Develocity Gradle Plugin
 on:
   push:
     branches: [ main ]
-    paths: [ 'convention-develocity-gradle-plugin/**' ]
+    paths: [ 'convention-develocity-gradle-plugin/**', '.github/workflows/convention-develocity-gradle-plugin-verification.yml' ]
```

Comment:
Not sure we should be too fine grained here, as a rename is likely to remain unnoticed.
I'd rather go with prefixed patterns

Reply:
That's true, but workflows are unlikely to change often and when they do, the impact of missing this is minimal. How would you configure the prefixed patterns?

Reply:
Maybe we could afford '.github/workflows/**'

Reply:
I agree with Jerome: making it simpler probably outweighs any advantage to only running the changed workflow.

Reply:
Do you know if GitHub exposes a variable for "path to current workflow" and if it could be used within an `on.push.paths:` condition? I'd really like to avoid running all workflows with `.github/workflows/**` when any workflow has changed. It would cause an explosion of unnecessary workflow runs which is where we used to be before adding these `on:` conditions in the first place.

Reply:
Not afaik

Reply:
@erichaagdev - I can't link as directly to it as I would like, but I see a `github.workflow_ref` that contains (as part of a longer string) the workflow file name: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/contexts#github-context

Would this solve your problem?

Reply:
I don't think so. We need the path to the workflow file exactly. Even if we had that though, I'm not sure it would work. Will need to test it.

Reply:
It turns out the `github` context isn't available in `on.push.paths`: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/contexts#context-availability

Reply:
In that case, I would lean towards Jerome's solution above (`.github/workflows/**`).

Reply:
Updated in b2157306b2e6ebf3746c0fb1ce9c0b62029fa0a0